scw/request_option.go

```diff
@@ -24,6 +24,13 @@ func WithAllPages() RequestOption {
 	}
 }
 
+// WithAPIURLRequest overrides the client API URL to the given URL in the request.
+func WithAPIURLRequest(apiURL string) RequestOption {
```

Comment:
Should we rename it to `WithRequestAPIURL` ? what do you think @QuentinBrosse

Reply:
The ability to clone a client would be nice and could replace this.

Reply:
`WithRequestAPIURL` is better yep.